src/components/Breadcrumbs/PageBreadcrumbs.tsx

```diff
@@ -0,0 +1,127 @@
+import styled from "@emotion/styled";
+import { useBreadcrumbAnalytics, BreadcrumbAnalytics } from "analytics";
+import { getVersionRoute, getCommitsRoute } from "constants/routes";
+import { size } from "constants/tokens";
+import { useGetUserPatchesPageTitleAndLink } from "hooks";
+import { shortenGithash } from "utils/string";
+import BreadCrumbs, { Breadcrumb } from ".";
+
+interface Props {
+  taskName?: string;
+  patchNumber?: number;
+  versionMetadata?: {
+    id: string;
+    revision: string;
+    project: string;
+    isPatch: boolean;
+    author: string;
+    projectIdentifier: string;
+  };
+}
+
+const PageBreadcrumbs: React.VFC<Props> = ({
```

Comment:
Would `VersionBreadcrumbs` or `VersionPageBreadcrumbs` make more sense? "Page" feels slightly generic since this component is really designed for a patch/version and its tasks.

Reply:
Yeah i'm a little torn over what to name this. Since its used by both Version and Task pages it seems odd to tie it to either. How about `VersionTaskPageBreadcrumb` ?